Specs/Scene/ModelExperimental/ModelRenderResourcesSpec.js

```diff
@@ -20,6 +20,7 @@ describe("Scene/ModelExperimental/ModelRenderResources", function () {
 
     expect(modelResources.model).toBe(mockModel);
     expect(modelResources.shaderBuilder).toBeDefined();
+    expect(modelResources.renderStateOptions).toBeDefined();
```

Comment:
To be a little more specific:
```suggestion
    expect(modelResources.renderStateOptions).toEqual({});
```